### ISSUE TITLE
Crash and Unable to Pair

### ISSUE THREAD

**Issue by livejamie** [winleafs/Winleafs#194]
I was doing screen mirroring last night and at some point it got stuck and crashed.

I left it overnight and there was no change.

I've tried reinstalling and repairing to no avail.

WInleafs can find my device but clicking pair doesn't do anything.

The official Android app seems to work fine.

[log.zip](https://github.com/winleafs/Winleafs/files/6826345/log.zip)

**Comment by StijnOostdam** [comment]
Hey, thanks for the report, I will try looking into it this weekend

**Comment by StijnOostdam** [comment]
What if you try to delete your existing settings located at `%appdata%/Winleafs`? Maybe your device changed IP and Winleafs does not handle that correctly

**Comment by livejamie** [comment]
I renamed that file to settings.old and it's still the same behavior. WInleafs can find my device immediately but clicking pair does nothing.

**Comment by StijnOostdam** [comment]
@livejamie Thanks for checking. I think the problem has something to do with the IP address of your panels, but I am not completely sure. I send you a test version via email (the email from your GitHub profile) with more logging, based on this commit https://github.com/winleafs/Winleafs/commit/58599c3d9ef83b684972c7f47ffda44df7af9185. Could you try that version without existing settings and post your logs here?

**Comment by livejamie** [comment]
Sure, feel free to toss it up on Google Drive

**Comment by StijnOostdam** [comment]
https://github.com/winleafs/Winleafs/releases/tag/untagged-0aaff73d9eaf4d7856b9 You can download it here. Just to be sure: uninstall WInleafs before installing this version

**Comment by livejamie** [comment]
That link gives me a 404 error

**Comment by StijnOostdam** [comment]
Does this link work for you? https://drive.google.com/file/d/167HhZvtMELBZyjtNJwr_e7gA_V8Ls7N_/view?usp=sharing

**Comment by livejamie** [comment]
Yeah

This is the log:

```
2021-07-20 15:11:35.2570|INFO|Winleafs.Wpf.Views.Setup.SetupWindow|Discovered following device: LEDE, IPs: 192.168.4.170,fe80::255:daff:fe59:f082%18, Port: 16021
2021-07-20 15:11:41.5516|INFO|Winleafs.Wpf.Views.Setup.SetupWindow|Selected following device: 192.168.4.170:16021
```

**Comment by livejamie** [comment]
Unless you're looking for something else from me?

**Comment by StijnOostdam** [comment]
Sorry, busy schedule, I haven't had time to check it yet.

**Comment by StijnOostdam** [comment]
@livejamie could you also send the expections that happen after those 2 lines?

**Comment by livejamie** [comment]
> @livejamie could you also send the expections that happen after those 2 lines?

Nothing else shows up, is there something I need to do to make them show up?

**Comment by StijnOostdam** [comment]
No, if there are exceptions they would be logged. So to reiterate:

- you select the device
- you hold the pair button on your device
- you click pair
- nothing happens

Correct?

Another question: which type of panels with which firmware version do you have?

**Comment by livejamie** [comment]
Correct,

I have the Nanoleaf Rhythm Edition, Triangles. 5.10.

It works in the Nanoleaf app on my Android phone.
